src/client/runner/automation/cursor.js

```diff
@@ -0,0 +1,74 @@
+import testCafeCore from '../deps/testcafe-core';
+import testCafeUI from '../deps/testcafe-ui';
+
+var domUtils = testCafeCore.domUtils;
+var CursorUI = window.top === window.self ? testCafeUI.CursorUI : testCafeUI.IframeCursorUI;
+
+
+//Static
+var x                   = 50;
+var y                   = 50;
+var currentActiveWindow = window.top;
+
+export default class Cursor {
```

Comment:
Class with static methods only is just a plain object. You can decalre `x`, `y`, `currentActiveWindow` as the object fields and they will be available via `this`